.github/workflows/frontend-workflow.yml

```
@@ -133,6 +133,36 @@ jobs:
           npm ci --loglevel verbose
         fi
 
+    - name: Validate specified deps not duplicated
+	if: ${{ inputs.custom_deps_not_dup == 'true' }}
```

NPM can be defined as an env var, with
```suggestion
	if: ${{ inputs.custom_deps_not_dup == 'true' }}
	env:
	   NPM: ${{ inputs.pm }}
```